test-framework/common/src/main/java/io/quarkus/test/common/TestResourceManager.java

```diff
@@ -84,10 +84,10 @@ public void stop() {
                 throw new RuntimeException("Unable to stop Quarkus test resource " + testResource, e);
             }
         }
-        ConfigProviderResolver cpr = ConfigProviderResolver.instance();
         try {
+            ConfigProviderResolver cpr = ConfigProviderResolver.instance();
             cpr.releaseConfig(cpr.getConfig());
-        } catch (IllegalStateException ignored) {
+        } catch (Throwable ignored) {
```

Comment:
`Exception` won't be generic enough here?

Reply:
This may not be nessesary any more, there were some smallrye class loading problems that resulted in Error's being thrown. I think they are fixed now as the CL always returns null from getParent, but I need to re-check.